news/feature-3368.md

```diff
@@ -0,0 +1 @@
+config: support `@version: current`
```

Comment:
just nitpicking note: when documented I would highlight that this is a keyword and not a string, so `@version: "current"` won't work.